test/cmdlineTests/standard_eWasm_requested/output.json

```diff
@@ -147,16 +133,6 @@
     (get_local $y)
 )
 
-(func $endian_swap
```

Comment:
Removal of a pure function - I would say this looks good even though I would need to see the yul code to actually find out what happened here.